ContactDetailsApi/V1/Domain/Enums.cs

```diff
@@ -27,6 +27,17 @@ public enum SubType
         home,
         work,
         other,
-        landline
+        landline,
+        mainNumber,
+        emergencyContact,
+        carer,
+        wife,
+        husband,
```

Comment:
For the sake of the bizarre world we find ourselves in, maybe it's best to also add a gender-neutral "spouse"

Reply:
Good point, I'll make sure to add this to MMH too